internal/controller/postgrescluster/apply_test.go

```diff
@@ -29,7 +29,6 @@ import (
 	"gotest.tools/v3/assert"
 	appsv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
-	v1 "k8s.io/api/core/v1"
```

Comment:
💯 `corev1` What a doozie! 👏🏻 

❓ We can enforce it with `importas`. What do you think of adding these rules?

```patch
diff --git .golangci.yaml .golangci.yaml
index fca0785dc..09a82f418 100644
--- .golangci.yaml
+++ .golangci.yaml
@@ -7,6 +7,7 @@ linters:
   enable:
     - gomodguard
     - gosimple
+    - importas
     - misspell
   presets:
     - bugs
@@ -28,6 +29,14 @@ linters-settings:
             reason: >
               k8s.io/kubernetes is for managing dependencies of the Kubernetes
               project, i.e. building kubelet and kubeadm.
+  importas:
+    alias:
+      - pkg: k8s.io/api/(\w+)/(v[\w\d]+)
+        alias: $1$2
+      - pkg: k8s.io/apimachinery/pkg/apis/(\w+)/(v[\w\d]+)
+        alias: $1$2
+      - pkg: k8s.io/apimachinery/pkg/api/errors
+        alias: apierrors
 
 run:
   build-tags:
```

I think we already comply with the first two rules. The last one requires a couple of changes in `internal/controller/postgrescluster/controller_ref_manager.go` and `internal/controller/postgrescluster/pgbackrest_test.go`.

Reply:
Yeah figured it would be good to go ahead and get that out of the way!

I like importas :+1: i'll add that in. Are there any other rules we can add?

Reply:
None that I know of right off. Very open to adding more later.